docs/it/trust_anchor_intermediari_endpoint.rst

```diff
@@ -0,0 +1,31 @@
+.. include:: ./common_definitions.rst
+
+
+
+Endpoint per Trust Anchor ed Intermediari
+-----------------------------------------
+Il Trust Anchor e i suoi Intermediari (federation_entity) DEVONO in aggiunta esporre al pubblico i seguenti endpoint:
+
+
+Fetch entity statement endpoint
++++++++++++++++++++++++++++++++
+
+Il recupero degli Entity Statement viene effettuato presso questo endpoint secondo le modalità definite all’interno di OIDC-FED “7.1. Fetching Entity Statements”.
```

Comment:
In tutti questi endpoint prego di fornire il link alla specifica OIDC Federation per un migliore navigazione

Reply:
fatto